discussions/no-customizing-classnames.md

```diff
@@ -1,58 +0,0 @@
-# No Customizing `className`'s
```

Comment:
I agree the current article is confusing given our current implementation.  We don't have to do it in this PR but we should talk about separation of concerns at some point.  The idea here is that the component is responsible for functionality and Fela is responsible for style, and modifying `className` without Fela violates that.